modules/graceful-restart.adoc

```diff
@@ -173,11 +173,11 @@ Check that the status for all nodes is `Ready`.
 ----
 NAME                           STATUS   ROLES                  AGE   VERSION
 ip-10-0-168-251.ec2.internal   Ready    control-plane,master   82m   v1.31.3
-ip-10-0-170-223.ec2.internal   Ready    control-plane.master   82m   v1.31.3
+ip-10-0-170-223.ec2.internal   Ready    control-plane,master   82m   v1.31.3
```

Comment:
It is for 4.19? `VERSION` uses `v1.32.3`, looks better. Better to hide the IP address.

Reply:
There is a separate PR actually merged yesterday that updates the version to 1.32.3. I need to rebase this PR and then you'll see it.